doc/whats_new/v0.21.rst

```diff
@@ -144,6 +144,14 @@ Support for Python 3.4 and below has been officially dropped.
   :func:`~model_selection.validation_curve` only the latter is required.
   :issue:`12613` and :issue:`12669` by :user:`Marc Torrellas <marctorrellas>`.
 
+- |Enhancement| :class:`~model_selection.BaseSearchCV` now allows for
```

Comment:
BaseSearchCV is not listed in `doc/modules/classes.rst` so this link won't work. Ordinarily we'd reference GridSearchCV and RandomizedSearchCV. you could also consider referencing the user guide rather than the example?